libultraship/libultraship/DefaultTexLoader.cpp

```diff
@@ -0,0 +1,31 @@
+#include "DefaultTexLoader.h"
+#include "Lib/stb/stb_image.h"
+#include "Archive.h"
+#include "File.h"
+#include "GlobalCtx2.h"
+#include "ResourceMgr.h"
+
+std::string normalize(std::string path) {
```

Comment:
Can this not be done with references to avoid copying it twice?

Reply:
It's only being copied once by the caller to create the argument. The return statement later calls the move-constructor to move the `path` variable to the newly created return value.

The bigger question is why we would need to transform / to \ at all. Preferably using / should work fine too, even on Windows.

Reply:
I think StormLib itself is now doing the normalization of slashes. @NEstelami